### ISSUE TITLE
License File

### ISSUE THREAD

**Issue by ABiomec** [delsys-inc/DelsysAPI-UnityExample#3]
In the UnityExample.cs I have to load the _Key_ & _License_. 

I have the Key, but I don't know the file I need to add in the License. 
Where can I find the _License_ file?
Does the _License_ string need to be loaded in a specific way?

I tried with the license.txt file from the folder [DelsysAPI.Core](http://data.delsys.com/DelsysServicePortal/api/index.html), 
but I get this error:

PipelineException: Error, product not licensed.
DelsysAPI.DelsysDevices.DeviceSource..ctor (System.String key, System.String license) (at <502adad0db5d4408b856e89de96733cd>:0)
DelsysAPI.DelsysDevices.DeviceSourcePortable..ctor (System.String key, System.String license) (at <502adad0db5d4408b856e89de96733cd>:0)
UnityExample.InitializeDataSource () (at Assets/UnityExample.cs:116)
UnityExample.Start () (at Assets/UnityExample.cs:70)

**Comment by joeybluebye** [comment]
I am having the same issue. Were you able to fix it?

**Comment by ABiomec** [comment]
Yes! I wrote directly to Delsys and they gave me the license file
